api-docs/client-interactions.md

```diff
@@ -0,0 +1,198 @@
+# Client Interactions
+
+The client interactions document direct encounters with the clients.
+
+Here is the list of the valid interaction types:
+
+- `clientInteractionCreated`,
+- `clientInteractionUpdated`,
+- `clientInteractionDeleted`,
+
+## Get a single Client Interaction
+
+### Request
+
+```http
+GET /api/clients/:id/interactions/:id
+```
+
+### Response
+
+```json
+{
+  "id": 1,
+  "title": "Nutrition Program - SNAP",
+  "interactionType": "walk-in",
+  "notes": "Application successfully filled out. Decision pending.",
+  "startTime": "2019-05-06T05:00:00.000Z",
```

Comment:
For some interactions, we might not have a start time or end time, but just a day and a duration.

What do you think would be the best way to represent that in the API?

Reply:
For the ones that do have a start and end time that could be handled on the front end and saved to the backend as just date and duration.

Reply:
Change added